cmd/deploy.go

```diff
@@ -89,19 +90,23 @@ func runDeployCmd(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		var err error
+		var e error
 		switch group.Kind {
 		case config.PackerKind:
 			// Packer groups are enforced to have length 1
-			moduleDir := filepath.Join(groupDir, string(group.Modules[0].ID))
-			err = deployPackerGroup(moduleDir)
+			subPath, err := modulewriter.DeploymentSource(group.Modules[0])
+			if err != nil {
```

Comment:
nit. `cobra.CheckErr` to save a couple of lines.

Reply:
Done (in several places)